### PR TITLE
Styled property fixes for Pedal and Ottava

### PR DESCRIPTION
**Pull request by mattmcclinch**
This PR addresses the following issues:

1. Pedal and Ottava lines are drawn too thin.
2. Some spanners do not follow style defaults when added to a range from the palette.
3. Style changes to Pedal Line Thickness and Pedal Line Style have no effect.

### CODE REVIEW

**Comment by anatoly-os** [comment]
Is there related issue? Issues are good place to discuss the solution and the issue itself.

**Comment by anatoly-os** [comment]
@mattmcclinch is there a related issue?

**Comment by mattmcclinch** [comment]
I don't think so. I had just noticed that there seemed to be some empty cells in the "Lines" palette. Upon closer look, I saw that they weren't empty, but just very faint. I did some investigation, and I found out why. After a bit of testing, I found a couple other related problems, and I realized what had to be changed in the code.

**Comment by anatoly-os** [comment]
@mattmcclinch ok, cool! Could you please add the issue with the description you gave above? It would be useful if someone finds the same issue and we will know the reason and how to fix it :)

**Comment by mattmcclinch** [comment]
Werner just fixed issue 1 with commit [f57973c](https://github.com/musescore/MuseScore/commit/f57973ce7ab49c400f94acb6d1f9afc88898bfdc).
But issues 2 and 3 still remain.

File: mscore/palette.cpp [hunk 1]
```diff
@@ -646,6 +646,7 @@ void Palette::applyPaletteElement(PaletteCell* cell)
                   for (int i = sel.staffStart(); i < endStaff; ++i) {
                         Spanner* spanner = static_cast<Spanner*>(element->clone());
                         spanner->setScore(score);
+                        spanner->initSubStyle(element->subStyleId());
```
Comment (mattmcclinch) [inline mscore/palette.cpp]: Solves issue 2.

File: libmscore/style.cpp [hunk 2]
```diff
@@ -1232,6 +1232,8 @@ const std::vector<StyledProperty> pedalStyle {
       { Sid::pedalHookHeight,                    Pid::BEGIN_HOOK_HEIGHT       },
       { Sid::pedalHookHeight,                    Pid::END_HOOK_HEIGHT         },
       { Sid::pedalBeginTextOffset,               Pid::BEGIN_TEXT_OFFSET       },
+      { Sid::pedalLineWidth,                     Pid::LINE_WIDTH              },
+      { Sid::pedalLineStyle,                     Pid::LINE_STYLE              },
```
Comment (mattmcclinch) [inline libmscore/style.cpp]: Solves issue 3.

**Comment by anatoly-os** [comment]
@mattmcclinch could you please rebase the PR?

**Comment by mattmcclinch** [comment]
I will just close this one, since it is completely out of date.

**Pull request closed without merging**